javascript/MaterialXView/source/viewer.js

```diff
@@ -79,6 +83,7 @@ export class Scene
         // Always reset controls based on camera for each load. 
         orbitControls.reset();
         this.updateScene(viewer, orbitControls);
+        viewer.getMaterial().updateMaterialAssignments(viewer);
```

Comment:
@jstone-lucasfilm, This makes it possible to load geometry first or corresponding materials first and get the same result. IMHO this is better than the desktop version
which uses the current "active" material to assign to loaded geometry. Also there is no concept of "active" material since all materials are accessible at the same time.